README.md

```diff
@@ -52,6 +52,11 @@ Then install what you just cloned.
 
 `pip install -e <path-to-platoon-folder>`
 
+### Attention
+Install ordering for dependencies:
+
+1. [NCCL](https://github.com/NVIDIA/nccl) should be installed first.
+2. [libgpuarray/ pygpu](https://github.com/Theano/libgpuarray) should be installed after the installation of NCCL.
```

Comment:
I found that if we install NCCL after install libgpuarray, then import theano.gpuarray in python, an error will be encountered. See [this issue](https://github.com/mila-udem/platoon/issues/85).